src/pixels.c

```diff
@@ -465,7 +625,8 @@ const char* drm_format_to_string(uint32_t fmt)
 	X(XRGB4444) \
 	X(ABGR4444) \
 	X(XBGR4444) \
-	X(RGB565)
+	X(RGB565)   \
+	X(BGR888)
```

Comment:
Missing RGB888 here. Maybe it would be nicer to have them between 8888 and 4444 as these are listed in a descending order of bpp.